sdcm/cluster_aws.py

```diff
@@ -280,9 +280,16 @@ def __init__(self, ec2_instance, ec2_service, credentials,
                               'Tagging node with {"keep": "alive"}',
                               cluster.TEST_DURATION)
                 tags_list.append({'Key': 'keep', 'Value': 'alive'})
+            if cluster.Setup.KEEP_ALIVE is True:
```

Comment:
No need for two "IFs" here. Please unify with test duration logic (when printing out the message just output the two values):
```
if cluster.TEST_DURATION >= 24 * 60 or cluster.Setup.KEEP_ALIVE:
...
```
### Good Python coding principles:
When checking boolean variable do not use "is" statement, "is" is isued for comparing reference IDs of two objects.
So the rule is:  use **==** when comparing **values** and **is** when comparing **identities**.

For boolean values, you shouldn't be doing comparisons at all. Instead of:

```
if x == True:
    # do something
```
write:
```
if x:
    # do something
```

Reply:
Fix. Some where read that other good pythonic way is to compare using 'is' for Boolen. Reread the PEP8 )